pkg/cmd/root/extension.go

```diff
@@ -4,20 +4,45 @@ import (
 	"errors"
 	"fmt"
 	"os/exec"
+	"strings"
 
 	"github.com/cli/cli/v2/pkg/extensions"
 	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/mgutz/ansi"
```

Comment:
So, I originally copied the logic for checking `gh` version logic exactly and modified only what was needed:

https://github.com/cli/cli/blob/215e4f440f56a0652e5c0f64614b64edcf1da1bc/internal/ghcmd/cmd.go#L169-L186

That said, I did update this to rely upon the `ColorScheme` logic because it honors disabling colors when applicable 👍